doc/layers.md

```diff
@@ -0,0 +1,92 @@
+# Layers
+
+Layers are an useful feature that lets you group renderization by context, and to also to obtain a way of pre rendering things, enabling for example the renderization in memory of parts of your game that don't change much, like a background for example, and by doing that, freeing resources for more dynamic content that needs to be rendered every loop cycle.
```

Comment:
>and to also to obtain a way of pre rendering things
as well as allow you yo pre-render things

> , enabling 
The phrase got a bit long, how about: `things. That enables, for example, ...`

Reply:
> like a background for example
like already implies it's an example; since you used for example just a few words ago I would remove it here to avoid repetition